docs/Actions/grant-user-role.md

```diff
@@ -0,0 +1,258 @@
+---
+id: grant-user-role
+title: Grant User Role
+sidebar_label: Grant User Role
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `Permission Scheme`
+
+## `Typical Use Cases`
+
+- Assign roles to new users on registration
+- Give access to resources for specific users
+- Temporary give access to users
+
+## `Related Actions`
+
+| Action Name                                             | Description                                                        |
+| ------------------------------------------------------- | ------------------------------------------------------------------ |
+| [Load Users](/docs/Actions/load-user.md)                | Can be used to grant roles to a single user or to a list of users. |
+| [User Registration](/docs/Actions/user-registration.md) | Can be used to grant roles to a newly created user.                |
+
+## `Input Parameter Reference`
+
+| Parameter        | Description                                                                                    | Supports Variables |
```

Comment:
Also, add "Required"( Yes/No) and Default(the default value ) as headers of this table.

Reply:
Done